dns/check-zone.sh

```diff
@@ -71,9 +75,17 @@ fi
 
 echo -n "Checking that the GCP dns servers for (${DOMAINS[*]})"
 echo    " serve up everything in our octodns config"
-./check-zone.py \
-    --config-file="${OCTODNS_CONFIG}" \
-    "${DOMAINS[@]/#/--zone=}"
+
+if [[ -n "${VALIDATE_ONLY}" ]]; then
+  ./check-zone.py \
+      --config-file="${OCTODNS_CONFIG}" \
+      "${DOMAINS[@]/#/--zone=}" "${VALIDATE_ONLY}"
```

Comment:
we can remove the branching around this and just do:
```suggestion
      "${DOMAINS[@]/#/--zone=}" "${VALIDATE_ONLY:+${VALIDATE_ONLY}}"
```

> ${parameter:+word}
> If parameter is null or unset, nothing is substituted, otherwise the expansion of word is substituted.

https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

Reply:
I looked for this option for sometime, finally I gave up and used this if/else block.. thanks for pointing me that out.. time to refresh scripting skills ;)